src/cli/mod.rs

```diff
@@ -194,15 +194,17 @@ fn generate_endpoint_subcommand(name: &str, endpoint: &Endpoint) -> clap::Comman
                         .action(ArgAction::Set),
                 ),
             ),
+            #[allow(clippy::needless_collect)] // intentionally evaluate fields now
             Post => command.subcommand(
                 clap::Command::new("post").args(
-                    &endpoint
+                    endpoint
                         .fields
                         .iter()
                         .filter_map(|(name, field)| generate_post_field_arg(name, field))
                         .collect::<Vec<clap::Arg>>(),
                 ),
             ),
```

Comment:
What's the reason for collecting here? As far as I can tell, `args(...)` immediately iterates over the input anyway.

Reply:
Indeed, this looks unnecessary. I had thought that `generate_post_field_arg` was doing more than it seems it is.